test_tools/fio/fio.py

```diff
@@ -37,14 +40,19 @@ def create_command(self, output_type=test_tools.fio.fio_param.FioOutput.json):
         return self.global_cmd_parameters
 
     def is_installed(self):
-        return self.executor.run("fio --version").stdout.strip() == self.fio_version
+        current_version = Version(
+            self.executor.run("fio --version").stdout.strip().removeprefix("fio-")
+        )
+        return current_version >= self.min_fio_version
```

Comment:
if fio is not installed at all then this will throw exception - Version() doesn't accept an empty string